.eslintrc.json

```diff
@@ -53,7 +53,11 @@
               "**/*.stories.tsx"
             ]
           }
-        ]
+        ],
+        "no-shadow": "off",
+        "@typescript-eslint/no-shadow": ["error"],
```

Comment:
it's a TSlint to ESlint bug and to fix it these configuration was added.
[StackOverflow link](https://stackoverflow.com/a/65768375/13964048)